### PR TITLE
company support now in master

### PR DESCRIPTION
**Pull request by pigmej**

### CODE REVIEW

File: company-nim.el [hunk 1]
```diff
@@ -58,22 +95,14 @@
...
-               (thing 'stop))
-           (and bounds
-                (if (eq face 'font-lock-comment-face)
-                    nil t)
-                (if (eq face 'font-lock-string-face)
-                    nil t)
-                (setq thing (buffer-substring-no-properties (car bounds)
-                                                            (cdr bounds))))
-           thing))))
+  "returns valid prefix for company"
```
Comment (reactormonk) [inline company-nim.el]: How about "checks if company-nim can complete here"?

File: company-nim.el [hunk 2]
```diff
@@ -58,22 +95,14 @@
...
+  (when (derived-mode-p 'nim-mode)
+    (let ((thing 'stop))
```
Comment (reactormonk) [inline company-nim.el]: Call it `result` or similar. `thing` is just bad imo.

Comment (pigmej) [inline company-nim.el]: it's a `thing` at point. I'm not sure if it's bad.

File: company-nim.el (hunk 1 above)
Comment (pigmej) [inline company-nim.el]: ok.

**Pull request merged**